app/models/concerns/bulkrax/has_matchers.rb

```diff
@@ -77,6 +84,10 @@ def object_name(field)
     # @return [Array] hyrax fields
     def field_to(field)
       fields = mapping&.map do |key, value|
+        if value['from'].instance_of?(Array)
+          value['from'] = value['from'].join(', ')
+        end
```

Comment:
without this change, `object` and `object_multiple` would both come back false. this happened because the field names had numbers at the end so the `include?` below would return false.